samples/ClassroomLabs/Modules/Library/Az.LabServices.psm1

```diff
@@ -1144,6 +1144,140 @@ function Get-AzLabVm {
     }
     end { }
 }
+# Decided on a separate function instead of adding a switch to Get-AzLabStudentVm as the concept of Status doesn't make sense.
+# It would create a situation when you could pass Status=Stopped and -Current at the same time.
+function Get-AzLabStudentCurrentVm {
+    [CmdletBinding()]
+    param()
+    try {
+        $callerEA = $ErrorActionPreference
+        $ErrorActionPreference = 'Stop'
+
+
+        if($IsLinux) {

```

Comment:
Where is $IsLinux defined?

Reply:
It should be a system variable defined by PS1 6.0+. Given that is not working for you I have changed the 'if' statement. Let's see if this one works ...

Reply:
I was testing on Mac just because it was convenient.  There is an $IsMacOs variable, too.  That's why the original code didn't work.  Sorry about that.  Maybe just return $null for Macs?  I think the system variable route is probably better as $env:os isn't guaranteed.